style.css

```diff
@@ -471,7 +476,17 @@ h1 {
     align-self: center;
   }
 
-  #about-me-text p {
+  #about-me {
+    background-image: url('./images/about-me-desktop.png');
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    height: 539px;
+    border-radius: 0;
+  }
```

Comment:
- [ ] It will be better 😃 if you increase the margin-top of the about text here as you see here the margin-top is less than the Figma Design and needed to increase to be like the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/2cc607fd-5d4b-4d1d-95be-97772cdde0fa)  |  ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/d0d8a275-cf06-455e-9cad-c5e5ea695993)  |